README.md

```diff
@@ -1 +1,44 @@
 
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+
+CC GLOBAL ENERGY SUMMIT WEBSITE
+
+
+## Built With
+-HTML
+-Javascript
+-CSS
+
+## Live Demo 
+
+[Live Demo Link](https://michaelmunavu83.github.io/capstone-project/)
+
+## Video Presentation
+
+[Video Presentation](https://www.loom.com/share/4c2be5b76f464a9dbed24566b1848dd5)
+
+## Acknowledgements
+
+Credit to Cindy Shin , the original designer
+
+
+## Authors
+
+👤 **Author1**
+
+- GitHub: [@githubhandle](https://github.com/MICHAELMUNAVU83)
+- Twitter: [@twitterhandle](https://twitter.com/MunavuMichael)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/michael-munavu-78703a218/)
+- 
+
+## Getting Started
+To get a local copy up and running  use gitclone or download the ZIP folder
```

Comment:
- [ ] [OPTIONAL] Here the reviewer means to add the actual code to run the project locally.For ex:-
```
git clone https://github.com/MICHAELMUNAVU83/capstone-project.git
cd capstone-project
```